docs/advanced-guides/cloud-in-a-box.md

```diff
@@ -309,3 +366,45 @@ This error means that your disk setup is broken. Use `cfdisk` and delete all par
 the system on which you want to install the Cloud in a Box image.
 
 With `lsblk` you can verify if the partitions are empty.
+
+## Running Cloud in a Box in a Virtual Machine
```

Comment:
I **really really really** appreciate to run CIAB virtualized because:

- we can reuse this to do integration tests with CIAB
- we can dramatically reduce the roundtrip time for ciab/node-image development steps 
- this help users which do not have a powerful extra machine but a powerful desktop system for to play with CIAB

Your documentation is a good start for that.
We should create a [node image](https://github.com/osism/node-image) for virtualized installs and describe the execution/usage here. 

Ar you interested to add the detailed steps here or peer with me to achieve this?

Reply:
I'm not sure what you would like to achieve here, but I would happily do it with you.

Reply:
Great :-) Let's discuss this in the next days.